README.md

```diff
@@ -126,7 +130,7 @@ As you are using a self-signed certificate you will need to import the created C
 docker build -f jetson-nano.Dockerfile -t "neuralet/smart-social-distancing:latest-jetson-nano" .
 
 # 3) Run Docker container:
-docker run -it --runtime nvidia --privileged -p HOST_PORT:8000 -v "$PWD":/repo -e TZ=`./timezone.sh` neuralet/smart-social-distancing:latest-jetson-nano
+docker run -it --runtime nvidia --privileged -p HOST_PORT:8000 -v "$PWD":/repo -e TZ=`./timezone.sh` --env-file .env neuralet/smart-social-distancing:latest-jetson-nano
```

Comment:
Adding this flag is optional right? What if the file does not exist, does it work? If not then I would prefer to remove the flag from these commands and just leave a note that when you want to use that feature you must add the flag to the command (as you did in the S3 section)

Reply:
You are right. I removed all the optional flag from the suggested commands.